gnovm/cmd/gno/transpile.go

```diff
@@ -233,18 +273,11 @@ func transpileFile(srcPath string, opts *transpileOptions) error {
 		return fmt.Errorf("write .go file: %w", err)
 	}
 
-	// check .go fmt, if `SkipFmt` sets to false.
-	if !flags.skipFmt {
-		err = transpiler.TranspileVerifyFile(targetPath, gofmt)
-		if err != nil {
-			return fmt.Errorf("check .go file: %w", err)
-		}
-	}
-
 	// transpile imported packages, if `SkipImports` sets to false
-	if !flags.skipImports {
-		importPaths := getPathsFromImportSpec(transpileRes.Imports)
-		for _, path := range importPaths {
+	if !flags.skipImports &&
+		!strings.HasSuffix(srcPath, "_filetest.gno") && !strings.HasSuffix(srcPath, "_test.gno") {
```

Comment:
Nit: shall we move this to a separated function called `isTestFile(srcPath) bool` or similar?

Reply:
Let's do this as part of https://github.com/gnolang/gno/pull/2201